cli/pipeline/run.go

```diff
@@ -125,39 +128,51 @@ func runPipelineStep(pipeline *PipelineDefinition, step *PipelineDefinitionStep,
 
 	containers := make(map[string]bool)
 
+	go func() {
+		time.Sleep(flags.StartTimeout)
+		if len(containers) < len(pod.Spec.Containers) {
```

Comment:
my original thought had been to leverage livenessprobe and readinessprobe to query container initiation but if the containers haven't even started yet, i can see how this check makes more sense. :+1: